resources/dicts/events/injury/general.json

```
@@ -3394,7 +3394,8 @@
                     "m_c"
                 ],
                 "scar": "m_c was scarred by a Clanmate that r_c encouraged {PRONOUN/m_c/object} to fight.",
-                "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after a fight with a Clanmate that r_c encouraged."
+                "reg_death": "m_c died from {PRONOUN/m_c/poss} wounds after a fight with a Clanmate that r_c encouraged.",
+                "lead_death": "died from {PRONOUN/m_c/poss} wounds after a fight with a Clanmate that r_c encouraged"
```

```suggestion
                "lead_death": "died from wounds sustained during a fight that r_c encouraged"
```
Okok so I *know* I just approved that, but rereading the full message, it's a bit clunky. It would now read "Ikeastar lost his first life when he died from wounds sustained during a fight that Stinkypaw encouraged". Yes, it loses a bit of the nuance, but there are like *five separate clauses* in this one sentence, and since we're constrained in how it starts, there's no way to break it up and make it feel more cohesive.

Honestly I'd suggest the same change with the regular death, but it's sliiightly less egregious over there